docs/install/docker-plugin.rst

```diff
@@ -0,0 +1,97 @@
+
+========================================
+Installing the Flocker Plugin for Docker
+========================================
+
+Before installing the Flocker plugin for Docker, you will need to have installed Flocker on some nodes, using the :ref:`node installation instructions <installing-flocker-node>`.
+
+The Flocker plugin for Docker requires access to the Flocker REST API, and therefore you will need to create an API user certificate and key for a user named ``plugin`` on each node, as described in the :ref:`generate-api` instructions.
+For example, you can use the ``flocker-ca`` command as below:
+
+.. prompt:: bash $
+
+   flocker-ca create-api-certificate plugin
+   scp ./plugin.crt root@172.16.255.251:/etc/flocker/plugin.crt
+   scp ./plugin.key root@172.16.255.251:/etc/flocker/plugin.key
+
+Upload these files to :file:`/etc/flocker/plugin.key` and :file:`/etc/flocker/plugin.crt` on the nodes where you want to run the Flocker plugin for Docker.
+
+You will also need to have Docker (at least version 1.8 or later) installed. For more information, see `the Docker installation instructions <https://docs.docker.com/>`_.
```

Comment:
this should say where Docker 1.8+ should be installed. on the nodes? on my local workstation? also, isn't that a prerequisite for installing Flocker that Docker is installed? so maybe the point of this section is to make it clear about the docker version dependency (1.8+)